OpenRA.Game/Game.cs

```diff
@@ -368,7 +368,9 @@ public static void InitializeMod(string mod, Arguments args)
 			Sound.StopVideo();
 
 			ModData = new ModData(Mods[mod], Mods, true);
-			ExternalMods.Register(ModData.Manifest);
+			var manifest = ModData.Manifest;
+			if (!ExternalMods.ContainsKey(ExternalMod.MakeKey(manifest)))
```

Comment:
A better fix for the exception is to change [ExternalMods.cs#L83](https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/ExternalMods.cs#L83) to `mods[ExternalMod.MakeKey(mod)] = mod;`.